app/Jobs/NotifyDeviceBatteryLowJob.php

```
@@ -0,0 +1,54 @@
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Queue\SerializesModels;
+
+class NotifyDeviceBatteryLowJob implements ShouldQueue
+{
+    use Dispatchable, InteractsWithQueue, Queueable, SerializesModels;
+
+    public function __construct() {}
+
+    public function handle(): void
+    {
+        $devices = Device::all();
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Models\Device' in method 'handle'. (Clean Code Rules)

(StaticAccess)